documentation/en/data-onboarding-visibility.md

```diff
@@ -0,0 +1,71 @@
+# Data Onboarding Visibility
+
+* [Introduction and background](#introduction-and-background)
+* [DDO information flow](#ddo-information-flow)
+* [Relevant message contents](#relevant-message-contents)
+* [Relevant builtin actor events](#relevant-builtin-actor-events)
+
+## Introduction and background
+
+**Direct Data Onboarding** (DDO) as defined in **[FIP-0076](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0076.md)** provides an optional data onboarding path that is both gas-efficient and paves a path toward eventual smart contract mediated onboarding mechanisms. The existing market actor and market actor mediated onboarding pathway remains largely unchanged but is now optional; it is anticipated that the gas savings alone will see a significant shift away from use of the market actor.
+
+Historically, a large amount of tooling was built around Filecoin that makes use of the market actor to quantify various data-related metrics. A shift in behaviour toward data onboarding that bypasses the market actor requires adaption in order to have continuity with the some of the data-related metrics being collected. This will continue to be true as Filecoin evolves to enable onboarding mechanisms mediated by smart contracts.
```

Comment:
```suggestion
Historically, a large amount of tooling was built around Filecoin that makes use of the market actor (f05) to quantify various data-related metrics. A shift in behaviour toward data onboarding that bypasses the market actor requires adaption in order to have continuity with the some of the data-related metrics being collected. This will continue to be true as Filecoin evolves to enable onboarding mechanisms mediated by smart contracts.
```